### PR TITLE
#1869 introduce +xunit

### PR DESCRIPTION
**Pull request by mximp**
Related to #1869.
This PR allows `+tests` meta which is treated similarly to `+junit`.
All `+junit` occurrences within EO core modules are replaced with `+tests`.

After this is merged - tasks to be created to perform replacement in all libraries, after which `+junit` can be removed completely from processing.

---

## PR-Codex overview
This PR replaces all mentions of "junit" with "tests" across multiple files in the project. 

### Detailed summary
- Replaced all instances of "junit" with "tests" in multiple files
- Added package declarations to some EO files in the `eo-runtime` and `eo-maven-plugin` directories
- Added `home` and `version` attributes to some EO files in the `eo-runtime` directory
- Added `alias` attribute to an EO file in the `eo-runtime` directory
- Removed a commented line in an EO file in the `eo-parser` directory
- Changed a value in an EO file in the `eo-maven-plugin` directory
- Changed file names in the `eo-maven-plugin` directory
- Changed XSL file names and content

> The following files were skipped due to too many changes: `eo-maven-plugin/src/main/resources/org/eolang/maven/pre/junit.xsl`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by mximp** [comment]
@yegor256 please verify and merge

File: eo-maven-plugin/src/main/resources/org/eolang/maven/pre/to-java.xsl [hunk 1]
```diff
@@ -160,7 +160,7 @@ SOFTWARE.
     <xsl:value-of select="eo:eol(0)"/>
     <xsl:apply-templates select="." mode="ctors"/>
     <xsl:apply-templates select="." mode="equals-and-hashCode"/>
-    <xsl:if test="//meta[head='junit'] and not(@parent)">
+    <xsl:if test="//meta[head='junit' or head='tests'] and not(@parent)">
```
Comment (Graur) [inline eo-maven-plugin/src/main/resources/org/eolang/maven/pre/to-java.xsl:163]: @mximp Nice to see this backward compatibility!

**Review by volodya-lombrozo — CHANGES_REQUESTED** [review]
@mximp Maybe we can add a puzzle to remove the `junit` keyword after all libraries have adopted the new word. Additionally, I suggest leaving one test that uses `+junit` to emphasize that we can still use it without breaking anything.

**Comment by yegor256** [comment]
@rultor merge

**Comment by mximp** [comment]
> @mximp Maybe we can add a puzzle to remove the `junit` keyword after all libraries have adopted the new word. Additionally, I suggest leaving one test that uses `+junit` to emphasize that we can still use it without breaking anything.

@volodya-lombrozo I was thinking about a puzzle here but opted to manually creating issue since this would involve updating all libs first.
I will link them all tother to make the sequence clear.

**Pull request merged**